README.md

```diff
@@ -25,6 +26,12 @@ Configuring the Docker images allows you to:
 
 To configure the Docker images, open up the `.env` file and make any necessary changes.  A brief description of the variables are below:
 
+### Global Variables
+  - PASS_PUBLIC_HOST: The hostname for the _public_ (shib protected) addresses.
```

Comment:
I don't fully understand these-

To give Fedora access outside of Shibboleth, I just gave it a separate port, so there wasn't a need to identify something like `PASS_DEV_HOST`.

Isn't the information contained in `PASS_PUBLIC_HOST` and `PASS_PUBLIC_PROTO` available via HTTP headers?  `X-Forwarded-For`, `X-Forwarded-Host`, `X-Forwarded-Proto`?

Reply:
It's needed for the fuseki to rewrite Fedora's URLs properly, and it can't use headers because it is changing _fedora_ URIs in requests and responses to _fuseki_.  The Fedora host name is not conveyed in requests to Fuseki.

Requests from "public" substitute a host of `${PASS_PUBLIC_HOST}` to and from `${FCREPO_HOST}:${FCREPO_PORT}`.  

Requests from "dev" substitute a host of `${PASS_DEV_HOST}:${PASS_DEV_PORT}` to and from `${FCREPO_HOST}:${FCREPO_PORT}`

I happen to use `localhost` for visual emphasis.  `localhost:8080` stands out more than `pass:8080` to me.  

So if you @emetsger use `pass` + ports for bypassing shib, you'd set `PASS_DEV_HOST` to `pass`.

We could keep it as-is,  make `pass` the the default, or remove `PASS_DEV_HOST` and put a steak in the ground that everybody must use `http://pass:<PORT>` for the non-shib case.

Reply:
I use pass+port for the dspace/fedora stuff, so if you can rebase this PR on that work and everything "just works", that's good enough for me!

Reply:
So, are these truly global variables?  Or are they just used for Fedora and Fuseki?

Reply:
It'll work.  At the moment, sparql queries are the only thing affected.  It might be worthwhile to make `pass` the default for dev, since that's the one value that will work both with docker-machine and docker-for-* or linux without having to change anything.